cargo-geiger/src/scan/default.rs

```diff
@@ -115,7 +115,7 @@ fn scan_to_report(
     root_package_id: PackageId,
     scan_parameters: &ScanParameters,
     workspace: &Workspace,
-) -> Result<Vec<String>, CliError> {
+) -> Result<(Vec<String>, u64), CliError> {
```

Comment:
A doc comment describing what this return type represents would be nice, alternatively a dedicated struct with a descriptive name and descriptive member names.

Reply:
Sounds good! I've created the `ScanResult` struct to be returned as the result, if that looks OK?

Reply:
LGTM